modules/cms/classes/ComponentBase.php

```diff
@@ -138,7 +138,7 @@ public function onRun()
     public function onRender()
     {
         if ($this->cacheTTL) {
-            return Cache::remember($this->getCacheKey(), now()->addMinutes($this->cacheTTL), function () {
+            return Cache::remember($this->getCacheKey(), $this->cacheTTL, function () {
```

Comment:
You'll need to do it like the following to support the pre and post Laravel 6 upgrade:

```php
$expires = is_integer($this->cacheTTL) ? now()->addSeconds($this->cacheTTL) : $this->cacheTTL;
```

Reply:
and then use $expires in Cache::remember